index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style/style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <title>Apple</title>
+  </head>
+  <body>
+    <nav class="nav-bar">
+      <button type="button" class="btn-item">
+        <i class="fa fa-apple" aria-hidden="true"></i>
+      </button>
+      <button type="button" class="btn-item">Store</button>
+      <button type="button" class="btn-item">Mac</button>
+      <button type="button" class="btn-item">iPod</button>
+      <button type="button" class="btn-item">iPhone</button>
+      <button type="button" class="btn-item">iPad</button>
+      <button type="button" class="btn-item">iTunes</button>
+      <button type="button" class="btn-item">Support</button>
```

Comment:
- [ ] The first eight elements should be links (to other sections or `#`) 

This looks great but kindly refer to the ([original page](https://web.archive.org/web/20140301004610/http://www.apple.com/)) to see that the navbar consists of a list of links in a list